subprojects/gradle/kotlin-dsl-support/src/main/kotlin/com/avito/kotlin/dsl/PropertyExtensions.kt

```diff
@@ -2,19 +2,24 @@
 
 package com.avito.kotlin.dsl
 
+import org.gradle.api.Project
 import org.gradle.api.file.RegularFile
 import org.gradle.api.internal.provider.Providers
 import org.gradle.api.provider.Property
 import org.gradle.api.provider.Provider
+import java.io.File
 
 /**
  * https://github.com/gradle/gradle/issues/2016#issuecomment-492598038
  */
 fun Property<RegularFile>.optionalIfNotExists(): Provider<RegularFile?> =
```

Comment:
optionalIfEmpty. Empty > Not exist

Reply:
yep, changed implementation and forgot to change signature

Reply:
it will be a breaking change, so let's call it `toOptional`

Reply:
not so breaking in fact, no clients uses it outside of this repo